src/api/app/models/bs_request.rb

```diff
@@ -1204,10 +1204,15 @@ def update_cache
     ).users.pluck(:user_id)
 
     user_ids << User.find_by_login!(creator).id
+
+    group_ids = Relationship.where(package_id: target_package_ids).or(
+      Relationship.where(project_id: target_project_ids)
+    ).groups.pluck('group_id')
     # rubocop:disable Rails/SkipsModelValidations
     # Skipping Model validations in this case is fine as we only want to touch
     # the associated user models to invalidate the cache keys
     User.where(id: user_ids).update_all(updated_at: Time.now)
+    Group.where(id: group_ids).update_all(updated_at: Time.now)
```

Comment:
[Nitpic] theres no need to do a second where query here, i think you can just do this?
```ruby
Relationship.where(package_id: target_package_ids).or(
  Relationship.where(project_id: target_project_ids)
).groups.update_all(updated_at: Time.now)
```

Reply:
Hm, good idea. However, groups is at the moment only a scope of the Relationship model (``where.not(group_id: nil)``). I will investigate.